ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,499 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7813
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.

Please also update the filename.